api/app/src/main/kotlin/packit/service/RoleService.kt

```diff
@@ -62,12 +51,22 @@ class BaseRoleService(
         return saveRole(createRole.name, permissions)
     }
 
+    override fun createUsernameRole(username: String): Role
```

Comment:
This renames and changes the semantics of `getUsernameRole` a bit.

I found it confusing that a method calls `get` has side-effects. I also don't think there's ever a valid use-case for returning an existing role, and it was opening up the door to someone creating a GitHub account whose username collides with an existing role, thereby gaining all of that roles permissions.